src/System.Data.SqlClient/ref/Configurations.props

```diff
@@ -1,9 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <BuildConfigurations>
+    <PackageConfigurations>
       netstandard;
       netfx;
+      uap;
```

Comment:
I don't think you need this change at all since your reference assembly isn't any different for `uap`: you can just let the `netstandard` (aka `netstandard2.0`) version apply.